### DISCUSSION TITLE
Perfomance when filter with large data

### DISCUSSION THREAD

**Discussion by josecao** [KevinVandy/material-react-table#17] [Q&A] [answered]
Hi, author 

I use filter in column with virtualization and column resizing (has 50 items), it is lag with filter data or out focus in input. You can try  the issue in storybook 

https://www.material-react-table.dev/?path=/docs/features-virtualization--enable-row-virtualization-with-column-resizing

**Comment by KevinVandy** [comment, marked as answer]
The reason that page is lagging is because it is rendering 3 examples on the page at the same time. The first example has rowVirtualization disabled on purpose to show the slower performance when it is off. Go to the canvas tab instead of the docs tab to just try out 1 example at a time, and you should see normal performance

https://www.material-react-table.dev/?path=/story/features-virtualization--enable-row-virtualization-with-column-resizing

**Reply by KevinVandy** [reply]
I'll remove the slow on purpose example to avoid confusion I guess

**Reply by josecao** [reply]
I am using canvas tabs, you will see it to lag if you scroll to line item 300 after toggle filter.

**Reply by KevinVandy** [reply]
Hmm I'll look into that more.
